README.md

```diff
@@ -1,5 +1,9 @@
-# ![nf-core/detaxizer](docs/images/nf-core-detaxizer_logo_light.png#gh-light-mode-only) ![nf-core/detaxizer](docs/images/nf-core-detaxizer_logo_dark.png#gh-dark-mode-only)
-
+<h1>
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="docs/images/nf-core-detaxizer_logo_dark.png">
+    <img alt="nf-core/detaxizer" src="docs/images/nf-core-detaxizer_logo_light.png">
+  </picture>
+</h1>
 [![GitHub Actions CI Status](https://github.com/nf-core/detaxizer/workflows/nf-core%20CI/badge.svg)](https://github.com/nf-core/detaxizer/actions?query=workflow%3A%22nf-core+CI%22)
```

Comment:
I assume here needs to be an empty line, cause the labels arent properly rendered.